exporter/collector/internal/integrationtest/fixtures.proto

```diff
@@ -22,4 +22,14 @@ message MetricExpectFixture {
   repeated google.monitoring.v3.CreateTimeSeriesRequest create_time_series_requests = 1;
   repeated google.monitoring.v3.CreateMetricDescriptorRequest create_metric_descriptor_requests = 2;
   repeated google.monitoring.v3.CreateTimeSeriesRequest create_service_time_series_requests = 3;
+  repeated SelfObservabilityMetric self_observability_metrics = 4;
+}
+
+message SelfObservabilityMetric {
+  // metric name
+  string name = 1;
+  // stringified value
+  string val = 2;
+  // stringified labels
+  map<string, string> labels = 3;
```

Comment:
Is this enough information or do we also want to capture type, description, etc.. On the extreme side, I could hack the OCAgent exporter and capture the full proto, but if we ever migrate these to OTel metrics API, this minimal set should be compatible.

Reply:
@sophieyfang for thoughts